plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveSplit.java

```diff
@@ -75,7 +77,8 @@ public HiveSplit(
             @JsonProperty("bucketConversion") Optional<BucketConversion> bucketConversion,
             @JsonProperty("bucketValidation") Optional<BucketValidation> bucketValidation,
             @JsonProperty("s3SelectPushdownEnabled") boolean s3SelectPushdownEnabled,
-            @JsonProperty("acidInfo") Optional<AcidInfo> acidInfo)
+            @JsonProperty("acidInfo") Optional<AcidInfo> acidInfo,
+            @JsonProperty("startingRowId") OptionalLong initialRowId)
```

Comment:
keep JSON key and parameter name consistent

Reply:
ok